### DISCUSSION TITLE
Body and Heading font not loaded on Safari iOS device, without any error

### DISCUSSION THREAD

**Discussion by Vinc26** [GetPublii/Publii#1852] [Help] [answered]
Hi there,

Here: [https://aurelielamour.com](https://aurelielamour.com)

It's very strange. Even if I have chose the Publii "Montserrat" font for Body and Heading font, ONLY on iOS it use Helvetica instead. It's like it has not loaded the Montserrat font. There is no error in the console.

But it works perfectly well on Safari or Chrome on macOS. And it work perfectly also on Android. It's only Safari iOS.

Could you test it on your side and tell me what you understand ?

<img width="1328" alt="Capture d’écran 2024-12-06 à 10 54 09" src="https://github.com/user-attachments/assets/c93c5667-5d45-41a0-8124-0a067fd8ef0f">
<img width="1330" alt="Capture d’écran 2024-12-06 à 10 54 42" src="https://github.com/user-attachments/assets/67c8f1d8-7422-4af3-859d-54514c283afb">

**Comment by Vinc26** [comment, marked as answer]
Oh, strange, it has been fixed simply adding this in htaccess:
```
RewriteCond %{HTTP_HOST} ^www\.(.+)$ [NC]
RewriteRule ^ https://%1%{REQUEST_URI} [L,R=301]
```

**Reply by bobmitro** [reply]
It's not a generic solution. Most likely, you are accessing the website with "www," but the domain in the server settings (Publii site settings) is configured without it, or vice versa.

**Reply by Vinc26** [reply]
Yes that's exactly that. It's a solution in my own case. Thanks for the clarification :)
